docs/development_guide.md

```diff
@@ -117,28 +117,40 @@ Refer to the [testing](advanced/testing.md) guide.
 
 In order to release new version you need to be owner of all the gems at RubyGems, ask one of the owners to add you before releasing. (Try `gem owners decidim`).
 
-Releasing new ***patch versions*** is quite easy, it's the same process whether it's a new version or a patch:
+Remember to follow the Gitflow branching workflow.
+
+### Create the release branch
 
-1. Checkout the branch you want to release: `git checkout -b VERSION-stable`
-1. Update `.decidim-version` to the new version number.
-1. Run `bin/rake update_versions`, this will update all references to the new version.
-1. Run `bin/rake bundle`, this will update all the `Gemfile.lock` files
-1. Run `bin/rake webpack`, this will update the JavaScript bundle.
-1. Update `CHANGELOG.MD`. At the top you should have an `Unreleased` header with the `Added`, `Changed`, `Fixed` and `Removed` empty section. After that, the header with the current version and link.
-1. Commit all the changes: `git add . && git commit -m "Prepare VERSION release"`
-1. Run `bin/rake release_all`, this will create all the tags, push the commits and tags and release the gems to RubyGems.
-1. Once all the gems are published you should create a new release at this repository, just go to the [releases page](https://github.com/decidim/decidim/releases) and create a new one.
-1. Finally, you should update our [Docker repository](https://github.com/decidim/docker) so new images are build for the new release. To do it, just update `DECIDIM_VERSION` at [circle.yml](https://github.com/decidim/docker/blob/master/circle.yml).
 
-If this was a **major version** release:
+### PRE-release
```

Comment:
Maybe we should call it [Release Candidate](https://en.wikipedia.org/wiki/Software_release_life_cycle#Release_candidate) as it's better to understand what we're talking about.